sample-app/src/main/java/com/joinforage/android/example/ui/pos/data/POSUIState.kt

```diff
@@ -9,7 +10,10 @@ data class POSUIState(
     val sessionToken: String = "<your_oath_or_session_token>", // <your_oath_or_session_token>,
     val merchantDetailsState: MerchantDetailsState = MerchantDetailsState.Idle,
     val tokenizedPaymentMethod: PaymentMethod? = null,
-    val balance: BalanceCheck? = null
+    val balance: BalanceCheck? = null,
+    val localPayment: PosPaymentRequest? = null,
+    val serverPayment: PaymentResponse? = null,
+    val paymentResponse: PaymentResponse? = null
```

Comment:
what's the difference between these two?

Reply:
Right now `serverPayment` is storing the result of calling `createPayment` and `paymentResponse` is storing the result of calling `capturePayment` but this naming doesn't convey that at all to the point that I even confused myself 😅 I'll rename to make it more apparent